lib/shared-ini/ini-loader.js

```diff
@@ -6,7 +6,7 @@ var path = require('path');
  * @api private
```

Comment:
Looks like you're making this public. Can you update the docs for this class as well?

If it is public, you'll want to create TypeScript definitions as well, and expose it on the `AWS` namespace.